### PR TITLE
suppress repack commands' error message

### PR DESCRIPTION
**Pull request by treelin611**
In Arkangel, the repack_dc file can be automatically generated from commands such as "set_clk" and "set_reset" in the PCF file. Therefore, some modifications are made in this PR to suppress the error messages for these two commands while reading pcf file.

### CODE REVIEW

File: libs/libpcf/src/io/pcf_reader.cpp [hunk 1]
```diff
@@ -57,12 +57,17 @@ int read_pcf(const char* fname, PcfData& pcf_data) {
           pcf_data.set_io_pin(io_id, pin_name);
         } else if (word[0] == COMMENT) {  // if it's a comment
           break;  // or ignore the full line comment and move on
+        } else if (word.find("set_clk") == 0 || word.find("set_reset") == 0) {
```
Comment (tangxifan) [inline libs/libpcf/src/io/pcf_reader.cpp:60]: My suggestion is to add a new option ``--reduce_error_to_warning`` to the read_pcf command.

**Review by tangxifan — CHANGES_REQUESTED** [review]

File: docs/source/manual/openfpga_shell/openfpga_commands/setup_commands.rst [hunk 2]
```diff
@@ -460,6 +460,10 @@ pcf2place
   .. option:: --no_time_stamp
 
     Do not print time stamp in output files
+  
+  .. option:: --reduce_error_to_warning
+
+    Reduce error to warning while reading commands in pcf file
```
Comment (tangxifan) [inline docs/source/manual/openfpga_shell/openfpga_commands/setup_commands.rst:466]: Add a warning message to alert users that they should be very cautious when enable this option. 

See an example here: https://docs.verilogtorouting.org/en/latest/vpr/command_line_usage/#cmdoption-vpr-strict_checks

File: libs/libpcf/src/io/pcf_reader.cpp [hunk 3]
```diff
@@ -58,11 +59,16 @@ int read_pcf(const char* fname, PcfData& pcf_data) {
         } else if (word[0] == COMMENT) {  // if it's a comment
           break;  // or ignore the full line comment and move on
         } else {
-          /* Reach unknown command, error out */
-          VTR_LOG_ERROR("Unknown command '%s'!\n", word.c_str());
-          num_err++;
-          break;  // and move onto next line. without this, it will accept more
-                  // following values on this line
+          if (reduce_error_to_warning) {
+            VTR_LOG_WARN("Unknown command '%s'!\n", word.c_str());
```
Comment (tangxifan) [inline libs/libpcf/src/io/pcf_reader.cpp:63]: Rephrase the message: 

```
Bypass unknown command '%s' 
```

**Review by tangxifan — CHANGES_REQUESTED** [review]
@treelin611 Good work. Can you add a testcase? Then it should be ready to merge. Thanks  for the contribution.

File: libs/libpcf/src/io/pcf_reader.cpp (hunk 3 above)
Comment (treelin611) [reply libs/libpcf/src/io/pcf_reader.cpp]: done!

**Pull request merged**